src/newrelic/processor_test.go

```diff
@@ -1471,11 +1497,31 @@ func runMockedCollectorHarvestLimitTest(t *testing.T, eventType string, agentLim
 
 func TestConnectNegotiateLogEventLimits(t *testing.T) {
 
-	runMockedCollectorHarvestLimitTest(t, "log_event_data", 90*12, 100, "agent limit smaller than collector")
-	runMockedCollectorHarvestLimitTest(t, "log_event_data", 110*12, 100, "agent limit larger than collector")
-	runMockedCollectorHarvestLimitTest(t, "log_event_data", 100*12, 100, "agent limit equal to collector")
-	runMockedCollectorHarvestLimitTest(t, "log_event_data", 0, 100, "agent limit == 0, collector != 0")
-	runMockedCollectorHarvestLimitTest(t, "log_event_data", 100*12, 0, "agent limit != 0, collector == 0")
+	// these tests exist for the log events (TestConnectNegotiateLogEventsLimit()) because at some point
+	// the collector would return the default limit (833) if 0 was passed as the log limit.  These tests
+	// exercise logic in the daemon to enforce the smaller agent value
+	runMockedCollectorHarvestLimitTest(t, "log_event_data", 90*12, 100, "agent log limit smaller than collector")
+	runMockedCollectorHarvestLimitTest(t, "log_event_data", 110*12, 100, "agent log limit larger than collector")
+	runMockedCollectorHarvestLimitTest(t, "log_event_data", 100*12, 100, "agent log limit equal to collector")
+	runMockedCollectorHarvestLimitTest(t, "log_event_data", 0, 100, "agent log limit == 0, collector != 0")
+	runMockedCollectorHarvestLimitTest(t, "log_event_data", 100*12, 0, "agent log limit != 0, collector == 0")
+
+}
+
+func TestConnectNegotiateCustomEventLimits(t *testing.T) {
+
+	runMockedCollectorHarvestLimitTest(t, "custom_event_data", 110*12, 100, "agent custom limit larger than collector")
+	runMockedCollectorHarvestLimitTest(t, "custom_event_data", 100*12, 100, "agent custom limit equal to collector")
+
+	// these tests exist for the log events (TestConnectNegotiateLogEventsLimit()) because at some point
```

Comment:
```suggestion
	// these tests exist for the custom events (TestConnectNegotiateLogEventsLimit()) because at some point
```

Reply:
It is a confusing comment but it is correct - it is attempting to explain why I had commented out the tests below this comment which for logging are still run.  I left them for reference but they could possibly be removed.

Reply:
Ah gotcha. Could we remove the commented out tests and simplify this comment to just reference the log tests above this?

Reply:
I made this change in commit  [12eb8f](https://github.com/newrelic/newrelic-php-agent/pull/584/commits/12eb8fb5b0f1a6a04dd5341fa11341a573c3a660).